
2022-10-24 14:58:57.667 - [Debug] [Auth] Attacker 81.68.253.52 trying to authenticate with 'publickey'
2022-10-24 14:58:57.667 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-24 14:58:57.668 - [Debug] [Auth] Attacker: 81.68.253.52 has so far made 1 attempts. Remaining: 5 attempts
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:1156403) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.167',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-24 15:00:00.952 - [Info] MITM Version: 2
2022-10-24 15:00:00.954 - [Info] Auto Access Enabled: true
2022-10-24 15:00:00.954 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-24 15:00:00.965 - [Info] SSH man-in-the-middle server for 10.0.3.167 listening on 10.0.3.1:1003
2022-10-24 15:00:01.275 - [Debug] [Connection] Attacker connected: 112.160.137.62 | Client Identification: SSH-2.0-Go
2022-10-24 15:00:01.839 - [Debug] [Connection] Attacker connected: 194.24.174.8 | Client Identification: SSH-2.0-Go
2022-10-24 15:00:02.569 - [Debug] [Auth] Attacker 112.160.137.62 trying to authenticate with 'none'
2022-10-24 15:00:02.569 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-24 15:00:02.668 - [Debug] [Auth] Attacker 194.24.174.8 trying to authenticate with 'none'
2022-10-24 15:00:02.668 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-24 15:00:02.972 - [Debug] [Auth] Attacker 194.24.174.8 trying to authenticate with 'publickey'
2022-10-24 15:00:02.974 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-24 15:00:02.974 - [Debug] [Auth] Attacker: 194.24.174.8 has so far made 1 attempts. Remaining: 5 attempts
2022-10-24 15:00:03.008 - [Debug] [Auth] Attacker 112.160.137.62 trying to authenticate with 'password'
2022-10-24 15:00:03.008 - [Debug] [Auto Access] Attacker: 112.160.137.62, Threshold: 1, Attempts: 1
2022-10-24 15:00:03.008 - [Debug] [Auto Access] Compromising the honeypot
2022-10-24 15:00:03.008 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-24 15:00:03.324 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-24 15:00:03.325 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-24 15:00:03.325 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.167
2022-10-24 15:00:03.329 - [Debug] [Connection] Attacker closed the connection
2022-10-24 15:00:03.374 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-24 15:00:03.407 - [Debug] [Connection] Attacker connected: 194.24.174.8 | Client Identification: SSH-2.0-Go
2022-10-24 15:00:03.534 - [Debug] [Connection] Attacker connected: 222.134.240.92 | Client Identification: SSH-2.0-Go
